providers/dev-guide.md

```diff
@@ -264,6 +264,15 @@ The logic in this class has
 quite strong assumptions on the data being stored. It assumes that it has the tree structure and each entity contains its kids as a nested hash.
 If you are able to achieve that structure in the `refresh_parser.rb`, you are halfway done. Otherwise, good luck :]
 
+### Defining Inventory
+
+Basically, refresh process consists of Collector, Persister and Parser classes (in most providers).
+When Collector loads data from external provider (like AWS), Persister defines structure of Inventory by InventoryCollection objects.  
+Inventory is then saved to VMDB (ManageIQ (also CFME) app database). Each inventory belons to certain ActiveRecord model.
+Parser maps collected data to these inventories.
```

Comment:
How about `Parser maps data collected from the provider to the common format defined by the ActiveRecord model` or something like that?